kafka-streams-serdes/src/test/proto/TestProtoRecord.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto3";
+
+package proto;
+
+message TestProtoRecord {
```

Comment:
Instead of creating a proto. just for the purpose of testing, can we use any of the built-in ones? E.g.: `Value` or `Timestamp` or `Duration`?